modular/manager/recover_scheduler.go

```diff
@@ -28,8 +28,8 @@ const (
 	MaxRecoveryTime  = 50
 
 	recoverInterval     = 30 * time.Second
-	verifyInterval      = 1 * time.Minute
-	verifyGVGQueryLimit = uint32(50)
+	verifyInterval      = 10 * time.Second
+	verifyGVGQueryLimit = uint32(100)
```

Comment:
should this be committed to develop branch?

Reply:
Yes